packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -1643,6 +1643,11 @@ function analyzePropertyChain(node, optionalChains) {
     return result;
   } else if (node.type === 'ChainExpression' && !node.computed) {
     const expression = node.expression;
+
+    if (expression.type === 'CallExpression') {
+      throw new Error(`Unsupported node type: ${expression.type}`);
```

Comment:
```suggestion
      throw new Error(`Unsupported node expression type: ${expression.type}`);
```
Maybe it gives a little bit more context on the console